config/base/env/config

```diff
@@ -10,7 +10,8 @@ REST_PORT=8080
 PROMETHEUS_PORT=9090
 TLS_HOSTNAME_OVERRIDE=
 TLS_PATH=/etc/tls
-NO_AUTH=false
+AUTH_DISABLE=false
+AUTH_IMPERSONATE=true
```

Comment:
Is this enabled by default? Doc above states that it is disabled by default?

Reply:
Ohh.. I missed this. 
Updated doc.